util/src/main/scala/com/scalableminds/util/tools/Converter.scala

```diff
@@ -10,6 +10,10 @@ object DefaultConverters {
       }
   }
 
+  implicit object StringToString extends Converter[String, String] {
```

Comment:
:grey_question:
(Why is this needed? Doesn't do much harm, just wondering…)

Reply:
The current implementation of Filters extracts the get parameters, and uses these Converters to type them. There wasn’t a string one before, and this implicit method was needed.

However, I guess we should stop using this method, and go for explicit get parameters, also allowing for standard swagger anotations. It may be that I can remove these converters then. I’ll have a look